packages/common/src/contexts/connection.tsx

```diff
@@ -244,15 +244,16 @@ export const sendTransactions = async (
   instructionSet: TransactionInstruction[][],
   signersSet: Account[][],
   awaitConfirmation = true,
-  commitment = 'singleGossip',
+  sendInSequence = false,
```

Comment:
Have you checked that there aren't other projects using this and changed their arguments too?

Reply:
Yes, it's used in governance only